File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/ProductoDAO.java
package pe.edu.pucp.softinv.dao;
import pe.edu.pucp.softinv.model.Producto.ProductoDTO;

public interface ProductoDAO {
    public Integer insertar(ProductoDTO producto);
    public ProductoDTO optenerPorId(Integer id);
    public Integer modificar(ProductoDTO producto);
    public Integer eliminar(Integer id);
}

File: C#/SoftInvPersistance/DAO/ProductoDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    interface ProductoDAO
    {
        int insertar(ProductoDTO producto);
        ProductoDTO obtenerPorId(int id);
        int modificar(ProductoDTO producto);
        int eliminar(ProductoDTO idProducto);
    }
}
